components/service/glean/src/main/java/mozilla/components/service/glean/private/CounterMetricType.kt

```diff
@@ -28,6 +28,26 @@ data class CounterMetricType(
 
     private val logger = Logger("glean/CounterMetricType")
 
+    /**
+     * ...
+     *
+     * @param value This is the new value for the counter.
+     */
+    fun set(value: Int) {
```

Comment:
As discussed [in the other PR](https://github.com/mozilla-mobile/fenix/pull/2749#discussion_r286814929), I don't think we need to change this. We can rely on the metrics being cleared after pings are sent :) If that doesn't happen, it's a bug on our end!